src/stcal/tweakreg/tweakreg.py

```diff
@@ -0,0 +1,342 @@
+from __future__ import annotations
+
+import math
+import warnings
+from pathlib import Path
+from typing import TYPE_CHECKING
+
+import astropy.units as u
+
+if TYPE_CHECKING:
+    import astropy
+    import gwcs
+from astropy.coordinates import SkyCoord
+from astropy.table import Table
+from tweakwcs.correctors import JWSTWCSCorrector
+from tweakwcs.imalign import align_wcs
+from tweakwcs.matchutils import XYXYMatch
+
+from stcal.alignment import wcs_from_footprints
+
+from .astrometric_utils import create_astrometric_catalog
+
+
+def _oxford_or_str_join(str_list):
+    nelem = len(str_list)
+    if not nelem:
+        return "N/A"
+    str_list = list(map(repr, str_list))
+    if nelem == 1:
+        return str_list
+    if nelem == 2:
+        return str_list[0] + " or " + str_list[1]
+    return ", ".join(map(repr, str_list[:-1])) + ", or " + repr(str_list[-1])
+
+
+_SQRT2 = math.sqrt(2.0)
+SINGLE_GROUP_REFCAT = ["GAIADR3", "GAIADR2", "GAIADR1"]
+_SINGLE_GROUP_REFCAT_STR = _oxford_or_str_join(SINGLE_GROUP_REFCAT)
```

Comment:
```suggestion
_SINGLE_GROUP_REFCAT_STR = ",".join(SINGLE_GROUP_REFCAT)
```
I don't think we need `_oxford_or_str_join` (and can remove the function with this change) here and can accept not having "or"s.